aggregator/src/aggregator/problem_details.rs

```diff
@@ -36,19 +36,30 @@ pub struct ProblemDocument<'a> {
     taskid: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     detail: Option<&'a str>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    collection_job_id: Option<String>,
 }
 
 impl<'a> ProblemDocument<'a> {
-    pub fn new(error_type: DapProblemType) -> Self {
```

Comment:
We should have doccomments explaining the distinction between `ProblemDocument::new` and `::new_dap` and explaining which should be used, when.